mobile/src/main/java/ch/epfl/sdp/mobile/ui/setting/SettingScreen.kt

```diff
@@ -75,7 +77,14 @@ fun SettingHeader(state: SettingScreenState, modifier: Modifier = Modifier) {
   ) {
     SettingPicture(state)
     Column(horizontalAlignment = Alignment.CenterHorizontally) {
-      Text(state.name, style = MaterialTheme.typography.h5)
+      Row(horizontalArrangement = Arrangement.Center) {
+        Text(state.name, style = MaterialTheme.typography.h5)
+        IconButton(
+            onClick = state::openEditProfileName,
+            modifier = Modifier.size(35.dp).testTag("editProfileName")) {

```

Comment:
Please don't use a `testTag` and prefer using the icon's content description (in this case, `strings.profileEditIcon`) in the tests.